pixoo/__init__.py

```diff
@@ -52,7 +53,7 @@ class Channel(IntEnum):
 
 class ImageResampleMode(IntEnum):
     PIXEL_ART = Image.NEAREST
-    SMOOTH = Image.ANTIALIAS
+    #SMOOTH = Image.ANTIALIAS
```

Comment:
This is being used in `examples.py`.
Why should this be commented out?

Reply:
as also in line 453 I forgot to remove the comments from the programming... i am still quite a beginner but i give my best :D